### PR TITLE
Post about organizing and maintaining knowledge bases

### PR DESCRIPTION
**Pull request by ABizzinotto**
**IMPORTANT: blog posts must be approved by two reviewers before they can be published, one of which must be a native English speaker.**

[Add any information about this post here. Leave the note above when opening the Pull Request.]

This post will link to the Tettra post (in progress).

### CODE REVIEW

File: _posts/2020-09-15-knowledge-bases.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+That's where a knowledge base comes in. At OmbuLabs we use [Tettra](https://app.tettra.co/) and we share our experience with it [here](link to Tettra article). But regardless of what tool you choose, here are some practices that will help you keep your knowledge base organized and updated.
+
+<!--more-->
+
+## Find a place to store all your documents
+
+Be it company policy pages, internal documents or external resources in any format, make sure you have a way of organizing them in a single place. Even if not possible to directly store video or audio files with your documents, you can always store them in a shared drive and link to them in a page inside your knowledge base. This way, every team member can easily find and access your resources as needed.
+
+## Create a structure for your content. And stickt to it
```
Comment (FionaDL) [inline _posts/2020-09-15-knowledge-bases.md:20]:
```suggestion
## Create a structure for your content. And stick to it
```

File: _posts/2020-09-15-knowledge-bases.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+Regardless of how you organize your knowledge base, make sure there is a guide to how to navigate it and add content to it and stick to this guide.
+
+## Have a clear procedure to add content
+
+One of the main points of having a knowledge base for us is to foster knowledge sharing and collaboration. We encourage all team members to suggest and add content to the knowledge base.
+
+But in order to make sure the structure of the knowledge base is respected and the content added is accurate and up to date, we developed a process that guides publishing new content pages. It contains steps to add new content, get it reviewed and approved, and published in the right place. It also has steps for team members to suggest content they believe is useful but are not able to write about so the best person for the job can take over and add the content.
```
Comment (FionaDL) [inline _posts/2020-09-15-knowledge-bases.md:32]:
```suggestion
But in order to make sure the structure of the knowledge base is respected and the content added is accurate and up to date, we developed a process that guides publishing new content pages. It contains steps to add new content, get it reviewed, approved, and published in the right place. It also has steps for team members to suggest content they believe is useful but are not able to write about so the best person for the job can take over and add the content.
```

File: _posts/2020-09-15-knowledge-bases.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+In order to make sure the process runs smoothly, everyone receives feedback and every suggestion is addressed, we have a center person to oversee the process.
```
Comment (FionaDL) [inline _posts/2020-09-15-knowledge-bases.md:34]:
```suggestion
In order to make sure the process runs smoothly, everyone receives feedback and every suggestion is addressed, we have a central person to oversee the process.
```

File: _posts/2020-09-15-knowledge-bases.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+Reviewing documents individually as they reach a certain "age" is better than creating a schedule to review everything in the entire knowledge base. You'll be able to keep your content up to date and, as the knowledge base grows, it might become impractical to dedicate time to review everything at once.
+
+## Take into account changes to the structure
+
+When creating a knowledge base, it is important to design a structure for it and stick to it. But as your team grows, things change and time passes, it's possible the structure of the knowledge base will need to change, with new categories being needed and old categories becoming obselete.
+
+It is important to take this into account. In order to address that, we created a process to suggest new categories as well. Any team member can make a suggestion, documenting why they think the new category is needed and how it'd be helpful. The suggestion is reviewed by the center person responsible for keeping the knowledge base organized and can be accepted or rejected.
```
Comment (FionaDL) [inline _posts/2020-09-15-knowledge-bases.md:48]:
```suggestion
It is important to take this into account. In order to address that, we created a process to suggest new categories as well. Any team member can make a suggestion, documenting why they think the new category is needed and how it'd be helpful. The suggestion is reviewed by the central person responsible for keeping the knowledge base organized and can be accepted or rejected.
```

File: _posts/2020-09-15-knowledge-bases.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+Having a knowledge base that easily integrates with the tool you use for internal communication is a great start. But also consider tools you use to create and share different kinds of documents and resources, as well as existing content you'd like to migrate or keep synced.
+
+For us, we knew from the start our knowledge base needed to be integrated with Slack, Google Docs, and GitHub. That is why we chose Tettra to power it.
+
+## Conclusion
+
+Regardless of the tool you choose to power your knowledge base, make sure it is easily accessible to your team, you have good procedures in place to add and remove content, as well as reviewing the structure, the content is kept up to date and the knowledge base integrates nicely with your other systems.
+
+Knowledge bases are particularly important for remote teams and are a great way to store your teams knowledge.
```
Comment (FionaDL) [inline _posts/2020-09-15-knowledge-bases.md:62]:
```suggestion
Knowledge bases are particularly important for remote teams and are a great way to store your team's knowledge.
```

**Review by FionaDL — COMMENTED** [review]
@ABizzinotto great article, only a few small changes.

**Comment by ABizzinotto** [comment]
Thanks @FionaDL! I've addressed your comments.

File: _posts/2020-09-15-knowledge-bases.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "Organizing and Maintaing a Team Knowledge Base"
```
Comment (lubc) [inline _posts/2020-09-15-knowledge-bases.md:3]:
```suggestion
title: "Organizing and Maintaining a Team Knowledge Base"
```

File: _posts/2020-09-15-knowledge-bases.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+Content becomes obsolete. Company policies change, technology changes, best practices change. It is important to make sure the content stored in your knowledge base stays fresh and up to date.
+
+In order to do that, define a schedule to review your content and make any needed changes. Make sure the schedule makes sense for your team and you have a way of notifying your knowledge base admin (or the person responsible for reviewing and updating content) when it's time to review a particular document.
+
+Reviewing documents individually as they reach a certain "age" is better than creating a schedule to review everything in the entire knowledge base. You'll be able to keep your content up to date and, as the knowledge base grows, it might become impractical to dedicate time to review everything at once.
+
+## Take into account changes to the structure
+
+When creating a knowledge base, it is important to design a structure for it and stick to it. But as your team grows, things change and time passes, it's possible the structure of the knowledge base will need to change, with new categories being needed and old categories becoming obselete.
```
Comment (lubc) [inline _posts/2020-09-15-knowledge-bases.md:46]:
```suggestion
When creating a knowledge base, it is important to design a structure for it and stick to it. But as your team grows, things change and time passes, it's possible the structure of the knowledge base will need to change, with new categories being needed and old categories becoming obsolete.
```

**Review by lubc — APPROVED** [review]
@ABizzinotto Great article, thanks!

**Pull request merged**